sass/oscailte/_variables.scss

```diff
@@ -5,7 +5,7 @@ $use-gridfix:       true;
 
 // Fonts & Font Families
 $serif: Georgia, 'Times New Roman', Times, serif !default;
-$sans-serif: "Helvetica Neue", Helvetica, Arial, sans-serif !default;
+$sans-serif: "Helvetica Neue", Noto Sans, Helvetica, Arial, sans-serif !default;
```

Comment:
You are adding a fallback font, that is not loaded on most systems, so this will not do anything on most systems. Only users that have manually installed `Noto Sans` and don't have `Helvetica Neue` will notice a change?

Reply:
Yup, this enhances the fallback stack on some Linux installs where "Helvetica" has some alignment issues (see: "documentation" in the navbar in the earlier screenshots).